### PR TITLE
NEO-1834 clicking on chevron should open select dropdown

### PR DESCRIPTION
**Pull request by yangAtSpoken**
[Link to updated components in Deploy Preview](https://deploy-preview-243--neo-react-library-storybook.netlify.app/?path=/story/components-select--basic-selects)

**Before tagging the team for a review, I have done the following:**
- [x] reviewed my code changes
- [x] ensured that all tests pass
- [x] updated the link at the top of this PR (or remove it if not applicable)
- [x] added any important information to this PR (description, images, GIFs, ect.)
- [x] tagged Matt if any visual changes have occurred
- [x] done an accessibility check on my work

bug fix

### CODE REVIEW

File: src/components/Select/InternalComponents/MultiSelect.tsx [hunk 1]
```diff
@@ -79,12 +79,12 @@ export const MultiSelect = () => {
         isOpen && "neo-multiselect--active",
       )}
     >
-      <span className="neo-multiselect-combo__header">
+      <span className="neo-multiselect-combo__header  neo-multiselect-combo__header--no-after">
```
Comment (joe-s-avaya) [inline src/components/Select/InternalComponents/MultiSelect.tsx:82]: nit: extra space
```suggestion
      <span className="neo-multiselect-combo__header neo-multiselect-combo__header--no-after">
```

File: src/components/Select/Select.cy.tsx [hunk 2]
```diff
@@ -1,7 +1,28 @@
...
+            cy.get("ul li").first().should("be.visible");
+          });
+      }
+    });
+  });
+});
+
 describe("Single Select Scrolling Tests", () => {
-  it("Last fruit in the list becomes visible with arow downs", () => {
+  it("Last fruit in the list becomes visible with arrow downs", () => {
```
Comment (joe-s-avaya) [inline src/components/Select/Select.cy.tsx:25]: good eye!

**Review by enrique-prado — CHANGES_REQUESTED** [review]
There are a couple of things that broke in the multiple select behavior:

1. After you make selections, you can no longer close the dropdown by clicking on the chevron. 
2. Keyboard users can no longer close the dropdown with the ESC key.

**Comment by matthewAvaya** [comment]
Overall it's looking good. For voice-over, the selection gets read twice on a lot of these. Please take a look at the screenshot for an example.
Screen Shot 2023-08-29 at 12 04 36 PM

**Comment by yangAtSpoken** [comment]
> Overall it's looking good. For voice-over, the selection gets read twice on a lot of these. Please take a look at the screenshot for an example. Screen Shot 2023-08-29 at 12 04 36 PM

@matthewAvaya If this is not a regression, let us create a new ticket to track this.

**Comment by yangAtSpoken** [comment]
> There are a couple of things that broke in the multiple select behavior:
> 
> 1. After you make selections, you can no longer close the dropdown by clicking on the chevron.
> 2. Keyboard users can no longer close the dropdown with the ESC key.

@enrique-prado For 1, you can no longer click chevron to close dropdown as chevron is part of the button now.  And the button right now does not toggle. 2. ESC key works for me. You have to show me how it is not working you.

**Comment by matthewAvaya** [comment]
> > Overall it's looking good. For voice-over, the selection gets read twice on a lot of these. Please take a look at the screenshot for an example. Screen Shot 2023-08-29 at 12 04 36 PM
> 
> @matthewAvaya If this is not a regression, let us create a new ticket to track this.

Thanks @yangAtSpoken for creating the new ticket.

**Comment by matthewAvaya** [comment]
> > There are a couple of things that broke in the multiple select behavior:
> > 
> > 1. After you make selections, you can no longer close the dropdown by clicking on the chevron.
> > 2. Keyboard users can no longer close the dropdown with the ESC key.
> 
> @enrique-prado For 1, you can no longer click chevron to close dropdown as chevron is part of the button now. And the button right now does not toggle. 2. ESC key works for me. You have to show me how it is not working for you.

Did this get resolved? The ESC key works for me as well to close the dropdown menu.

**Comment by enrique-prado** [comment]
@matthewAvaya Regarding #2, I can only repro the ESC issue if I use the mouse to make selections. That's because using the mouse does not give the select component focus.  But since keyboard users use the keyboard to make selections and in that scenario the ESC key works I'm going to approve this PR.  If you can open a ticket for the mouse click / focus issue that would be great. thanks.

**Review by enrique-prado — APPROVED** [review]

File: src/components/Select/utils/useDownshift.ts [hunk 3]
```diff
@@ -28,11 +28,13 @@ const DownshiftWithComboboxProps = (
     id: selectId,
     stateReducer: (_, actionAndChanges) => {
       const { changes, type } = actionAndChanges;
+      const isOpen = changes.isOpen ? !(disabled || loading) : false
```
Comment (enrique-prado) [inline src/components/Select/utils/useDownshift.ts:31]: Nice!

**Comment by matthewAvaya** [comment]
> @matthewAvaya Regarding #2, I can only repro the ESC issue if I use the mouse to make selections. That's because using the mouse does not give the select component focus. But since keyboard users use the keyboard to make selections and in that scenario the ESC key works I'm going to approve this PR. If you can open a ticket for the mouse click / focus issue that would be great. thanks.

I see, good catch. Okay, I'll create a new Jira for this. In this case, I'll go ahead and approve the PR as well. Thanks!

**Review by matthewAvaya — APPROVED** [review]

**Pull request merged**